.rubocop.yml

```diff
@@ -65,3 +65,148 @@ RSpec/MessageSpies:
 RSpec/AnyInstance:
   Exclude:
     - 'spec/scnnr/connection_spec.rb'
+
+RSpec/ContextWording:
+  Prefixes:
+    - when
+    - with
+    - without
+    - and
+
+RSpec/MultipleMemoizedHelpers:
+  Max: 7
```

Comment:
Since we use a lot of `let` in this project, I chose 7 to avoid most of our use cases (default is 5) and applied code changes to the rest.

For more details about this change by Rubocop-Rspec:
      - https://github.com/rubocop-hq/rubocop-rspec/pull/863
      - https://github.com/rubocop-hq/rubocop-rspec/issues/862
      - https://thoughtbot.com/blog/lets-not